rdkcompositor.cpp

```diff
@@ -204,7 +207,8 @@ namespace RdkShell
         return success;
     }
 
-    bool RdkCompositor::createDisplay(const std::string& displayName, const std::string& clientName, uint32_t width, uint32_t height)
+    bool RdkCompositor::createDisplay(const std::string& displayName, const std::string& clientName,
+        uint32_t width, uint32_t height, bool virtualDisplayEnabled, uint32_t virtualWidth, uint32_t virtualHeight)
```

Comment:
It doesn't look like virtualDisplayEnabled, virtualWidth, and virtualHeight are honored when passed into createDisplay.  We should call `enableVirtualDisplay(virtualDisplayEnabled)` and `setVirtualResolution (virtualHeight, virtualHeight)`.